Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -69,16 +77,21 @@ public class BTShopperInsightsClient {
                 isVenmoRecommended: venmo?.recommended ?? false,
                 isEligibleInPayPalNetwork: payPal?.eligibleInPayPalNetwork ?? false || venmo?.eligibleInPayPalNetwork ?? false
             )
-            return self.notifySuccess(with: result)
+            return self.notifySuccess(with: result, for: experiment)
         } catch {
-            throw self.notifyFailure(with: error)
+            throw self.notifyFailure(with: error, for: experiment)
         }
     }
 
     /// Call this method when the PayPal button has been successfully displayed to the buyer.
     /// This method sends analytics to help improve the Shopper Insights feature experience.
-    public func sendPayPalPresentedEvent() {
-        apiClient.sendAnalyticsEvent(BTShopperInsightsAnalytics.payPalPresented)
+    /// - Parameter buttonRank: Optional:  The position of the button in the list of available payment methods.
```

Comment:
oh yea that's a great callout ~ not entirely sure but will check with product to see what they prefer

Reply:
update: had a conversation with product yesterday and they let us know we can use the `payment_methods_displayed` param [here](https://internal.paypalinc.com/fptigov/tags/dictionary) instead of `buttonRank` but there is an existing issue with finding the associated fpti tag, so this is on pause for now